<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="SDEVPCSVU7112" name="com.test.regression.cub.testscripts.MyAccount" tests="1" failures="0" timestamp="20 Sep 2016 05:29:07 GMT" time="89.569" errors="1">
  <testcase name="ChngContInfoInvalidLastName" time="89.569" classname="com.test.regression.cub.testscripts.MyAccount">
    <error type="java.lang.AssertionError" message="The following asserts failed:

 Failed for LastName with Special Characters, 
 Failed for LastName with Numbers">
      <![CDATA[java.lang.AssertionError: The following asserts failed:

 Failed for LastName with Special Characters, 
 Failed for LastName with Numbers
	at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:37)
	at com.test.regression.cub.testscripts.MyAccount.ChngContInfoInvalidLastName(MyAccount.java:701)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:75)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:46)
]]>
    </error>
  </testcase> <!-- ChngContInfoInvalidLastName -->
</testsuite> <!-- com.test.regression.cub.testscripts.MyAccount -->
